### PR TITLE
add a little note about component i18n

### LINKED ISSUE #949: i18n - Is it possible to translate the components?
If so are there any documentation on how can I do it? It's the only thing preventing me from adopting this awesome toolkit!

### PR DESCRIPTION
**Pull request by giladgray**

### CODE REVIEW

File: packages/core/src/docs/typography.md [hunk 1]
```diff
@@ -83,6 +83,9 @@ that contains text.
 
 @## Internationalization
 
+I18n in Blueprint is straightforward. React components expose props for customizing any strings;
+use the library of your choice for managing internationalized strings.
```
Comment (adidahiya) [inline packages/core/src/docs/typography.md]: protip: use `jsx-use-translation-function` to partially enforce this https://github.com/palantir/tslint-react

Comment (giladgray) [inline packages/core/src/docs/typography.md]: wow that is great! do you want me to include a link to that here? seems somewhat tangential.

Comment (adidahiya) [inline packages/core/src/docs/typography.md]: I leave it up to you. probably too off-topic

Comment (giladgray) [inline packages/core/src/docs/typography.md]: agreed too off-topic

**Pull request merged**